.gitignore

```diff
@@ -3,3 +3,4 @@ Cargo.lock
 .envrc
 default.nix
 shell.nix
+.idea
```

Comment:
this need not be in the commit. Depends on dev workflow, ide.
Can we worked around by using gobal .gitignore on dev system

Reply:
Removed.